src/structures/Message.js

```diff
@@ -25,6 +25,11 @@ class Message {
          * The ID of the chat the message was sent in
          */
         this.chatID = threadID
+        /**
+         * @type {object}
+         * The full message payload instagram is sending (forwarded by the dilame/instagram-private-api)
+         */
+        this.data = data
```

Comment:
I wasn't sure how to name this variable:
- data
- payload
- messagePayload

I decided for `data` because in the constructor you named it `data` as well.

Reply:
how about `Object.defineProperty(this, "data", { value: data })` instead of `this.data = data`?

Reply:
@Snowflake107 why would you want to use that syntax instead? I tried to use the same pattern @Androz2091 used inside [Message.js](https://github.com/Androz2091/insta.js/blob/39790feb41c4e7efdc5f41ae5d10f78064131d79/src/structures/Message.js). The question was only concerning the naming of the instance variable, so either:

```js
this.data = data
// VS
this.payload = data
// VS
this.messagePayload = data
```

Or am I missing something here?

Reply:
it should work as usual but with `this.data = something`, it just floods the object (not clean) and since it is a raw data, better to use `Object.defineProperty`

Reply:
you can also lock the property using `writable: false` descriptor

Reply:
Ok thanks just read this [SO post](https://stackoverflow.com/questions/10105824/when-do-you-use-object-defineproperty) and now I get what you mean. With `Object.defineProperty` you could make the propert readonly for example, which would make sense, I agree!